source/Calamari.ConsolidateCalamariPackages/ConsolidatedPackageCreator.cs

```diff
@@ -56,7 +56,7 @@ Dictionary<string, string[]> GroupByPlatform(IEnumerable<SourceFile> filesForPac
                     .GroupBy(f => f.Platform)
                     .ToDictionary(
                         g => g.Key,
-                        g => g.Select(f => f.Hash).OrderBy(h => h).ToArray()
+                        g => g.Select(f => Path.Combine(f.Hash, f.FullNameInDestinationArchive.Replace('/', Path.DirectorySeparatorChar))).OrderBy(h => h).ToArray()
```

Comment:
We should hardcode to use `/` as it's supported on windows, but `\` is not supported on unix-based systems